app/multitenant/dynamo_collector.go

```diff
@@ -141,6 +156,25 @@ func NewDynamoDBCollector(
 		}
 	}
 
+	var memcacheClient *memcache.Client
+	if memcachedHost != "" {
+		var memcacheServers memcache.ServerList
+		memcacheClient = memcache.NewFromSelector(&memcacheServers)
+		memcacheClient.Timeout = memcachedTimeout
+		err := updateMemcacheServers(memcachedHost, memcachedService, &memcacheServers)
+		if err != nil {
+			// REVIEWER: Undecided whether this should exit or not. Thoughts?
+			log.Errorf("Could not set memcache servers: %v", err)
```

Comment:
Whats the behaviour if you run this thing and it can't talk to the servers?

What we want: a client that can be instantiated without throwing errors if it cannot resolve/contact the servers, but which then throws errors on individual operations.